services/web/client/source/class/osparc/store/Store.js

```diff
@@ -212,6 +203,13 @@ qx.Class.define("osparc.store.Store", {
       } else {
         this.setCurrentStudyId(null);
       }
+    },
+
+    addGroupToCatalog: function(group) {
```

Comment:
can we directly use the ``Resources`` class? if constructing the object ``params`` seems cumbersome, maybe we can come up with an easy refactor..